browser/src/map/handler/Map.KeyboardShortcuts.ts

```diff
@@ -175,6 +175,7 @@ keyboardShortcuts.definitions.set('default', new Array<ShortcutDescriptor>(
 
     new ShortcutDescriptor('text', 'keydown', 0, 'F2', null, null, null),
     new ShortcutDescriptor('text', 'keydown', 0, 'F3', '.uno:ExpandGlossary', null, null),
+    new ShortcutDescriptor('text', 'keydown', Mod.CTRL, 'F3', null, null, null),
```

Comment:
shouldn't be before the plain F3 ? so in case we have also ctrl we will use that rule before